data/meson.build

```diff
@@ -8,6 +8,7 @@ portal_sources = files(
   'org.freedesktop.portal.Account.xml',
```

Comment:
Commit message header should be e.g. `Add Clipboard interface` (describing the change). The commit message body already gives more details so IMO good enough.